courses/models.py

```diff
@@ -229,6 +230,15 @@ def current_price(self):
             return None
         return latest_version.price
 
+    @property
+    def course_lineup(self):
+        """Gets the CoursesInProgram subpage if associated with this program"""
+        if self.page:
```

Comment:
I know we follow that style in other places too but can we change it like that from now ? 

@property
```
    def course_lineup(self):
        """Gets the CoursesInProgram subpage if associated with this program"""
        if not self.page:
            return None:
        
        child = self.page.get_children().type(CoursesInProgramPage).first()
        if child:
           return child.specific
```

Reply:
Sure.